test/src/tests/stats.test.js

```diff
@@ -0,0 +1,55 @@
+import {
+  CLIENT_URL,
+  registerNewUser,
+  registerDandiset,
+  uniqueId,
+} from '../util';
+import * as homePage from '../pages/homePage';
+
+describe('home page stats', () => {
+  beforeAll(async () => {
+    await Promise.all([
```

Comment:
Would it be helpful to have a helper like
```
async function loadPage(pageUrl) {
  return Promise.all([
    // TODO: Handle the case where CLIENT_URL ends with a "/"
    page.goto(`{CLIENT_URL}/#{pageUrl}`),
    // wait until there are no active network connections for 500ms
    page.waitForNavigation({ waitUntil: 'networkidle0' }),
  ]);
}
```

Right here, it could be used as simply `await loadPage('/')`.

Reply:
Quite possibly. I really don't like the `Promise.all` syntax that's required to effectively use `waitForNavigation`, so a helper would make that less painful. However I'm still not 100% sold that this `networkidle0` thing is a good idea, I just wanted to try it out since puppeteer offered it.
I think a helper would still be useful though, I will add one.